src/Forms/PresentedOptionsField.php

```diff
@@ -82,6 +89,11 @@ public static function getSelectTypes()
         ];
     }
 
+    public function performReadonlyTransformation()
+    {
+        return $this->setReadonly(true);
```

Comment:
Usually these methods are immutable - might be worth returning a clone of $this as well